proxy/Cargo.toml

```diff
@@ -39,6 +39,9 @@ utils = { path = "../libs/utils" }
 metrics = { path = "../libs/metrics" }
 workspace_hack = { version = "0.1", path = "../workspace_hack" }
 
+x509-parser = "0.13.2"
+substring = "1.4.5"
```

Comment:
I don't think we need this.

Reply:
I added it bc the compiler complained, I'll check again.

Reply:
done